scipy/stats/tests/test_morestats.py

```diff
@@ -1144,9 +1144,7 @@ def test_ppcc_max_bad_arg(self):
     def test_ppcc_max_basic(self):
         np.random.seed(1234567)
         x = stats.tukeylambda.rvs(-0.7, loc=2, scale=0.5, size=10000) + 1e4
-        # On Python 2.6 the result is accurate to 5 decimals. On Python >= 2.7
-        # it is accurate up to 16 decimals
```

Comment:
On two different systems these are only equal to 11 decimals (e.g. I tried 12 and it failed) same with the two below